root/static/scripts/edit/URLCleanup.js

```diff
@@ -5550,7 +5549,6 @@ function testAll(tests: $ReadOnlyArray<RegExp>, text: string) {
   return false;
 }
 
-// $FlowIssue[incompatible-type]: Array<mixed>
 const CLEANUP_ENTRIES: Array<CleanupEntry> = Object.values(CLEANUPS);
```

Comment:
Is `Array<CleanupEntry>` still needed here even if `CLEANUP` has `+[type: string]: CleanupEntry`?

Reply:
Looks like it's not needed in Flow 0.197.0 at least. I'm not sure if it's needed under the new inference mode though and it can't hurt to keep, so I'll leave it alone for now.